src/ilastik-packager.imjoy.html

```diff
@@ -145,7 +145,10 @@
   if(spec.source){ // && spec.source.startsWith('./')
     const src = spec.source.split(':')[0] // e.g. ./mynetwork:UNet
     files.push(src)
-    exportSpec.source = getFileName(src)
+    // exportSpec.source = getFileName(src)
+    // HACK: leave field in exportSpec unchanged, bioimageio.spec currently only
+    // loads it like this
+    exportSpec.source = spec.source
```

Comment:
Oh, right, this is the correct implementation, I will make another PR to also leave the dependencies unchanged